src/views/Home.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import styled from "styled-components";
+import QuestionCard from "../components/QuestionCard";
+import Hero from "../components/Hero";
+
+export default function Home(props) {
+  const list_questionID = ['1', '2']
+
+  return <div>
+    <Hero />
+    <ContentWrapper>
+      {list_questionID.map((questionID) =>
+        <div style={{ borderBottom: "1px solid #6C718A" }} key={questionID}>
```

Comment:
Let's replace all inline style with styled components

Reply:
fix in commit aa9c275